server/routes/serviceProviderReferrals/interventionProgressPresenter.test.ts

```diff
@@ -87,6 +87,60 @@ describe(InterventionProgressPresenter, () => {
       expect(presenter.sessionTableRows).toEqual([])
     })
 
+    describe('when there are multiple sessions', () => {
+      it('they must be ordered correctly', () => {
```

Comment:
_nit_ when a regression occurs, I find it useful if the test defines what "correct" is, so the test failure also explains what the correct should be